cmake/kokkos_enable_devices.cmake

```diff
@@ -40,7 +40,6 @@ ELSE()
 ENDIF()
 KOKKOS_DEVICE_OPTION(OPENMP ${OMP_DEFAULT} HOST "Whether to build OpenMP backend")
 
-
```

Comment:
Any reason to change this file?

Reply:
Leftovers from the rebase,  will be removed.